minerl/herobraine/hero/handlers/agent/observations/lifestats.py

```diff
@@ -46,6 +47,18 @@ def xml_template(self) -> str:
         return str("""<ObservationFromFullStats/>""")
 
 
+class _IsAliveObservation(LifeStatsObservation):
+    """
+    Handles is_alive observation. Initial value is True (alive)
+    """
+
+    def __init__(self):
+        keys = ['is_alive']
+        super().__init__(hero_keys=keys, univ_keys=keys,
+                         space=spaces.Box(low=False, high=True, shape=(), dtype=np.bool),
+                         default_if_missing=True)
```

Comment:
Is it true that if the observation is missing the value should be true, rather than false?

Reply:
Hmm, I think so. My assumption is that it's in case we haven't requested that this key be returned as an observation from Minecraft at all, in which case it's a better bet that the agent is alive rather than dead. Another way to see this is that default_if_missing for _LifeObservation is the max life rather than 0, which seems consistent with is_alive being true rather than false.

Reply:
Fair. In that case, looks good to me!